src/overlays/actors/ovl_En_Peehat/z_en_peehat.cpp

```diff
@@ -516,7 +516,7 @@ void EnPeehat_Flying_StateRise(EnPeehat* pthis, GlobalContext* globalCtx)
 		if(SkelAnime_Update(&pthis->skelAnime) || pthis->animTimer == 0)
 		{
 			//! @bug: overwrites number of child larva spawned, allowing for more than MAX_LARVA spawns
-			pthis->unk2FA = 0;
+			pthis->larvaCount = 0;
```

Comment:
Same here, again it's not necessary for this PR but might as well deal with it if possible.

Reply:
not sure but that wuld be a bug ofc the original game?

Reply:
pehat has an issue with going too high and a few other things too, so it can be revisited

Reply:
According to the comment it is, but it could very well not be accurate. Anyways it's totally up to you to go above and beyond and figure that out, no worries if you do not want to, the rest looks good from what I can tell.